src/index.html

```diff
@@ -11,7 +11,135 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+
+        <label class="label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="label">
+          Full date of birth:
+          <input type="date" name="birth">
+        </label>
+
+        <label class="label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="termOfAgree"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+
+        <label class="label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="15"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="label">
+          Do you love cats?
+          <input type="radio" name="y"> Yes
+          <input type="radio" name="n"> No
+        </label>
```

Comment:
Do not wrap all by label. When I click on text - Yes become selected

https://user-images.githubusercontent.com/61904995/223121495-2bd59567-6f5a-49fc-969c-70ef3804425b.mp4

Reply:
please check my update